Dockerfile

```diff
@@ -17,4 +17,7 @@ LABEL description="Materialize dbt adapter GitHub Action"
 COPY profiles.yml.template /profiles.yml
 COPY entrypoint.sh /entrypoint.sh
 
+# Copy poetry files into the container
+COPY poetry.lock pyproject.toml ./
```

Comment:
@bobbyiliev not sure on the best location for these files. I was treating these like the `profiles.yml.template` as an example of what it might look like in the repo being used. Though true the COPY should probably be conditional.